src/bp/core/services/action/local-action-server.ts

```diff
@@ -59,10 +59,18 @@ const _validateListActionsRequest = async (req: Request, res: Response, next: Ne
   next()
 }
 
+interface RunActionResponse extends Response {
+  send: (body: ActionServerResponse) => RunActionResponse
+}
+
+interface RunActionRequest extends Request {
+  body: { actionArgs: any; actionName: string; botId: string; token: string; incomingEvent: IO.IncomingEvent }
+}
```

Comment:
In `routers/sdk/utils.ts`, there's a generic interface for Request. You could also add one for the response, this way you can reuse the same interface you have elsewhere, ex: `TypedRequest<RunActionProps>`

```js
export interface TypedRequest<T> extends Request {
  body: T
}
```

Reply:
Done in e083b37